### PR TITLE
added Alligator swag to data.json

### PR DESCRIPTION
**Pull request by olessiap**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

hey there! I added the alligator info to data.json. I also signed up to write for the platform to get the swag. I'll update here once that happens!

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -8,6 +8,15 @@
     "dateAdded": "2018-02-18T06:03:00.000Z",
     "tags": ["clothing", "device"]
   },
+  {
+    "name": "Alligator.io",
+    "difficulty": "hard",
+    "description": "Write a blog post on frontend development topics, get some cool Alligator stickers",
+    "reference": "https://alligator.io/write-for-us/",
+    "image": "https://pbs.twimg.com/profile_images/622200384455442432/hf71Z251_400x400.png",
+    "dateAdded": "2018-12-15T06:03:00.000Z",
```
Comment (vikaspotluri123) [inline data.json]: This should be `2018-12-09T07:08:00.000Z` since we (devSwag) were notified then

Comment (olessiap) [reply data.json]: @vikaspotluri123, hello! I fixed the timestamp. Can you please tell me where you see it? I see that the issue was opened 7 days ago, is that what you're referring to? 
:)

Comment (vikaspotluri123) [reply data.json]: Yep, that's exactly right!

File: data.json [hunk 2]
```diff
@@ -8,6 +8,15 @@
     "dateAdded": "2018-02-18T06:03:00.000Z",
     "tags": ["clothing", "device"]
   },
+  {
+    "name": "Alligator.io",
+    "difficulty": "hard",
+    "description": "Write a blog post on frontend development topics, get some cool Alligator stickers",
```
Comment (aslafy-z) [inline data.json]: Let's add some ending ponctuation to keep consistency with other swag items

Comment (olessiap) [reply data.json]: @aslafy-z whoops, wow good eye. I fixed it!

**Comment by vikaspotluri123** [comment]
Thanks again @olessiap!

**Pull request merged**